### PR TITLE
More types of alcohol

### PR DESCRIPTION
**Pull request by Brytenwalda**
#### Summary
```SUMMARY:  Content "Several new types of alcohol."```

#### Purpose of change
This change is small but surely very significant for all of us alcoholics out there. It will add variety to our travels.

#### Describe the solution
I added several types of whisky and fortified wines (sherry and madeira).
I also added it to proper item groups and made it spawn in mansions. 

#### Testing
Everything should be working just fine. It's a very small change.

### CODE REVIEW

**Comment by curstwist** [comment]
Make sure your put them into some item_groups so they can spawn.

I'd also request you add them to this specific flask item_group:
https://github.com/CleverRaven/Cataclysm-DDA/blob/d560dfc47119623f97c2e17332238169a5e98fe0/data/json/itemgroups/food_service.json

```{
    "id": "flask_liquor",
    "type": "item_group",
    "subtype": "distribution",
    "items": [
      [ "flask_hip", 100 ],
      { "item": "whiskey", "prob": 40, "container-item": "flask_hip" },
      { "item": "vodka", "prob": 30, "container-item": "flask_hip" },
      { "item": "gin", "prob": 20, "container-item": "flask_hip" },
      { "item": "rum", "prob": 20, "container-item": "flask_hip" },
      { "item": "tequila", "prob": 20, "container-item": "flask_hip" },
      { "item": "brandy", "prob": 10, "container-item": "flask_hip" },
      { "item": "moonshine", "prob": 5, "container-item": "flask_hip" }
    ]
  }
```

**Comment by LaVeyanFiend** [comment]
Needs a [linting](http://dev.narc.ro/cataclysm/format.html)

**Comment by Brytenwalda** [comment]
> 
> 
> Make sure your put them into some item_groups so they can spawn.
> 
> I'd also request you add them to this specific flask item_group:
> https://github.com/CleverRaven/Cataclysm-DDA/blob/d560dfc47119623f97c2e17332238169a5e98fe0/data/json/itemgroups/food_service.json
> 
> ```
>     "id": "flask_liquor",
>     "type": "item_group",
>     "subtype": "distribution",
>     "items": [
>       [ "flask_hip", 100 ],
>       { "item": "whiskey", "prob": 40, "container-item": "flask_hip" },
>       { "item": "vodka", "prob": 30, "container-item": "flask_hip" },
>       { "item": "gin", "prob": 20, "container-item": "flask_hip" },
>       { "item": "rum", "prob": 20, "container-item": "flask_hip" },
>       { "item": "tequila", "prob": 20, "container-item": "flask_hip" },
>       { "item": "brandy", "prob": 10, "container-item": "flask_hip" },
>       { "item": "moonshine", "prob": 5, "container-item": "flask_hip" }
>     ]
>   }
> ```

Thanks! I'll add them soon.

**Comment by Night-Pryanik** [comment]
I'm ok with madeira and sherry, but do we really need two more varieties of whiskey? We already have whiskey in the game after all.

**Comment by Brytenwalda** [comment]
> I'm ok with madeira and sherry, but do we really need two more varieties of whiskey? We already have whiskey in the game after all.

We already have many different types of wine, why can’t we have more types of whiskey?

**Comment by FuelType-Memes** [comment]
Is it possible to use snippet_category to add different varieties of liquids?

**Comment by curstwist** [comment]
> 
> 
> Is it possible to use snippet_category to add different varieties of liquids?

I've tried this using tea varieties.  It sort of works but not really.  You can spawn them in with the snippet but once you stack bottles of the teas they all take on a single snippet.

File: data/json/items/comestibles/alcohol.json [hunk 1]
```diff
@@ -1070,6 +1070,168 @@
...
+    "description": "Fortified wine from Madeira.  A true Royal Navy drink.",
+    "price": 2400,
+    "material": "alcohol",
+    "volume": "750 ml",
+    "flags": [ "EATEN_COLD" ],
+    "freezing_point": -22,
+    "phase": "liquid",
+    "charges": 7,
+    "fun": 15
+  },
```
Comment (DaviBones) [inline data/json/items/comestibles/alcohol.json]: Again, please have these two items inherit from sherry.

**Comment by Kodiologist** [comment]
Don't forget to capitalize proper nouns (e.g., "irish" should be "Irish"), and using the word "bottle" in descriptions isn't a great idea, because the descriptions apply to the liquids no matter what container you put them in.

**Comment by kevingranade** [comment]
I'd rather merge this and bolt on the inheritance stuff later than just can it.

**Comment by Jovus** [comment]
Did this also involve changing the appropriate recipes to account for the new alcohol types? I'm thinking specifically of strong mixed alcohol and ethanol, which lead to ether.

**Comment by DaviBones** [comment]
Looks like no. I added your point to a new issue that summarizes the changes that still ought to be made to these new items.

**Pull request merged**